packages/styles/global/variables.scss

```
@@ -44,6 +44,10 @@
   --color-contextual-warning-regular: #ffc900;
   --color-contextual-warning-light-1: #ffda52;
   --color-contextual-warning-light-2: #fff4cc;
+  --color-contextual-info-dark-1: #0A6171;
+  --color-contextual-info-regular: #0D8296;
+  --color-contextual-info-lighT-1: #10A2BC;
+  --color-contextual-info-lighT-2: #E3F9FD;
```

suggestion: following the others variables patterns, the last T could be in lower case

```suggestion
  --color-contextual-info-light-1: #10A2BC;
  --color-contextual-info-light-2: #E3F9FD;
```

---

Fix here: https://github.com/juntossomosmais/venice/pull/608/commits/4bedada112054e7c0dc2bc5bf215dbc821b0ef70